src/commands/Checkin.ts

```diff
@@ -175,35 +179,93 @@ export default class Checkin extends Command {
    */
   private static async generateQRCodeURL(event: PortalEvent, expressCheckinURL: URL) {
     // Create the QR code. This library is very undocumented, so we'll make it simpler to read.
```

Comment:
I'd like for this function to either generate just the QR code as a URL or the widescreen slide as a URL (we can pass this in as a boolean `slide: boolean`, so we should move the slide drawing code to a new function (name it `createQRSlide()`)